Plugins/FastJet/include/ActsPlugins/FastJet/Jets.hpp

```diff
@@ -19,70 +21,80 @@
 
 namespace ActsPlugins::FastJet {
 
-/// Default jet definition: Anti-kt with a radius of 0.4
-const fastjet::JetDefinition DefaultJetDefinition =
-    fastjet::JetDefinition(fastjet::antikt_algorithm, 0.4);
+/// Common class for jets
+class Jet {
+ public:
+  explicit Jet(const Acts::Vector4& fourMom) { m_fourMomentum = fourMom; }
+
+  /// @brief Get the jet 4-momentum
+  /// @return the jet 4-momentum as an Acts::Vector4
+  Acts::Vector4 getFourMomentum() const { return m_fourMomentum; }
```

Comment:
```suggestion
  Acts::Vector4 fourMomentum() const { return m_fourMomentum; }
```
nitpick: usually we omit the `get`